aruco_ros/src/aruco_detector.cpp

```diff
@@ -207,19 +210,120 @@ class ArucoSimple
     return "Unknown code";
   }
 
+  int get_home_from_id(int id){
+    char str[100];
+
+    for (unsigned i = 0; i < num_markers_in_list; ++i){
+      if (marker_id[i] == id){
+        return i;
```

Comment:
shouldn't this return 1? If the marker is the first in the list it will return 0 as if it wasn't there